src/App.vue

```diff
@@ -565,11 +562,11 @@ export default {
       // Draft Applications are always in good standing
       this.setGoodStanding(true)
 
-      // store NR Number if present
+      // save local NR Number if present
       this.localNrNumber = nameRequest.nrNumber || null
 
       // store Legal Name if present
-      this.setEntityName(nameRequest.legalName || null)
+      if (nameRequest.legalName) this.setEntityName(nameRequest.legalName)
```

Comment:
This is better -- leave the initial store value alone if we don't set an entity name here.